docs/user_guide/dashboard_api.rst

```diff
@@ -75,8 +75,6 @@ Running ``nvflare dashboard -h`` shows all available options.
     -p PORT, --port PORT  port to listen
     -f FOLDER, --folder FOLDER
                             folder containing necessary info (default: current working directory)
-    -i DASHBOARD_IMAGE, --dashboard_image DASHBOARD_IMAGE
-                            container image for running dashboard
     --passphrase PASSPHRASE
```

Comment:
this only updates the documentation, have you made the changes in code?

Reply:
Codes were changed in #916 .